backend/decompme/settings.py

```diff
@@ -138,5 +138,7 @@
 
 COMPILER_BASE_PATH = BASE_DIR / "compilers"
 LOCAL_FILE_DIR = BASE_DIR / "local_files"
-ASM_OBJECTS_PATH = LOCAL_FILE_DIR / "assemblies"
-COMPILATION_OBJECTS_PATH = LOCAL_FILE_DIR / "compilations"
+
+USE_SANDBOX = True
```

Comment:
I think we should use the django environ method of setting this, so having `env('USE_SANDBOX')` as the value here and adding a line for it with the default value being True in the object on line 10. And then in the .dev in the repo, we can also specify it as True which makes it easy to disable locally (just change the .env file like you do for other settings)